### PR TITLE
Azeez/feature/nav bar new

### PR DESCRIPTION
**Pull request by nomanoff**
nav is ready

image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -11,27 +11,57 @@ body{
...
 
+a{
+    text-decoration: none;
+    color: #000;
+}
+
+a:hover{
+    color: #268a35;
+}
+
```
Comment (wahidustoz) [inline style.css]: Are you sure that you want all the anchor tags to have this attributes? What will you do if you want to make a new anchor somewhere with other style?

File: index.html [hunk 2]
```diff
@@ -13,9 +13,22 @@
 
     <div class="main">
         <div class="nav-bar">
-            
+            <div class="website-name">
```
Comment (wahidustoz) [inline index.html]: Bad name

File: index.html [hunk 3]
```diff
@@ -13,9 +13,22 @@
...
+                <h2>codwiz.net</h2>
+            </div>
+
+            <div class="nav-bar-home">
```
Comment (wahidustoz) [inline index.html]: change `nav-bar-home` to `nav-bar`

**Review by wahidustoz — COMMENTED** [review]
- Add font-weight 500 for nav-bar items. 
`<link href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@300;400;500;700&display=swap" rel="stylesheet">`

**Review by wahidustoz — COMMENTED** [review]
# nav-bar Shadow
- remove border-bottom from nav-bar
- add elevation to the bottom, probably same elevation as the cards

image

**Review by nomanoff — COMMENTED** [review]
- enhanced navbar
- better naming
image

File: index.html [hunk 4]
```diff
@@ -12,27 +12,62 @@
...
+                <h2>Recents Posts</h2>
+            </div>
+
+            <ul class="bar-nav">
 
-                <i class="far fa-bookmark fa-lg"></i>
+                <li class="bar-nav-item">
+                    <i class="far fa-bookmark fa-lg"></i>
+                </li>
+
```
Comment (wahidustoz) [inline index.html:65]: bar is wrong name

**Review by wahidustoz — APPROVED** [review]

**Pull request merged**